MCAntiMalware-Core/src/main/java/optic_fusion1/antimalware/notifications/impl/LinuxNotifSender.java

```diff
@@ -44,8 +45,8 @@ public LinuxNotifSender() {
       libNotify = new DefaultJLibnotifyLoader().load();
       libNotify.init("AntiMalware");
       notificationImage = File.createTempFile("notificationImage", "").toPath();
-      Files.copy(Main.class.getResource("/index.jpg").openStream(), notificationImage,
-              StandardCopyOption.REPLACE_EXISTING);
+      Files.copy(Objects.requireNonNull(Main.class.getResource("/index.jpg")).openStream(), notificationImage,
```

Comment:
This will always exist, so checking if it's null doesn't make sense. I can't really think of a reason it wouldn't.